integration/common.go

```diff
@@ -47,8 +47,8 @@ var (
 
 func initImages(imageListFile string) {
 	imageList = ImageList{
-		Alpine:           "docker.io/library/alpine:latest",
-		BusyBox:          "docker.io/library/busybox:latest",
+		Alpine:           "docker.io/library/alpine:latest", // TODO: have alpine on ghcr.io as well.
```

Comment:
@mxpv or @estesp Not really sure I can push Alpine to containerd org's GitHub Packages due to permissions. Can you?

Reply:
Yep, let me check that.

Reply:
Try ghcr.io/containerd/alpine:3.14.0